src/registrar/admin.py

```diff
@@ -192,6 +203,138 @@ def response_change(self, request, obj):
                     % obj.name,
                 )
             return HttpResponseRedirect(".")
+
+        if GET_SECURITY_EMAIL in request.POST:
```

Comment:
Untangling this will be a bit of a headache, is this something you'd want me to help you on? I've dealt with this particular function enough to understand how it works pretty well so just separating all of this out shouldn't take me too long

Reply:
delete for me please